packages/components/NativeRadioButton/src/NativeRadioButton.tsx

```diff
@@ -0,0 +1,70 @@
+/** @jsx withSlots */
+import { ViewProps } from 'react-native';
+import { compose, buildProps, mergeProps, UseSlots, withSlots, LayoutTokens } from '@fluentui-react-native/framework';
+import { ensureNativeComponent } from '@fluentui-react-native/component-cache';
+
+const nativeRadioButtonName = 'NativeRadioButton';
+
+const NativeRadioButtonView = ensureNativeComponent('RadioButtonView');
+
+export type NativeRadioButtonProps = {
+  /**
+   * Radio button title, default title is set to "Button".
+   */
+  title?: string;
+  /**
+   * Whether the radio button is selectable or not, true by default.
+   */
+  enabled?: boolean;
+  /**
+   * Current state of the radio button. There are two states:
+   * true -> On
+   * false -> Off
+   * false by default.
+   */
+  state?: boolean;
+  /**
+   * Unique identifier for each radio button.
+   */
+  buttonKey?: string;
+  /**
+   * On press event block
+   */
+  onPress?: () => void;
+};
```

Comment:
Nit: Could we change these to be closer to `RadioButton's` properties? AKA, `content` instead of `title`, and `disabled` instead of `enabled`? The idea being to keep the API as similar as possible to the original RadioButton to make it easier for clients?

Reply:
I know we talked a bit about this offline, my rationale was since this API is tailored to native platforms, it's appropriate to follow the native naming conventions.

Reply:
We talked offline more. We thought about merging the changes here into the existing `@fluentui-react-native/radiobutton` package so that it's a JS component on win32 and a native component on macOS. There are a few issues with that approach, namely that some non-optional props from the JS control are not implemented in the macOS native component yet, and other props would be no-ops. We weren't sure what's a better experience for clients: Using the same package but having some props not work on macOS vs win32, or having two different packages altogether. 

The conclusion we came to was to try both, and collect client feedback. My `MenuButton` component shares the package with win32 and has many props as non-optional. Amanda's `NativeRadioButton` component is a separate package with different props altogether. Both are about to be consumed by different clients so we can see the pros and cons of each approach.

Reply:
Thanks for the write-up